### ISSUE TITLE
twisted.web.http docstring indicates chunked transfer is not supported

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4911]
|<img alt="andy's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| andy reported|
|-|-|
|Trac ID|trac#4911|
|Type|defect|
|Created|2011-02-24 22:49:24Z|

From the module docstring list of future plans:

" - Accept chunked data from clients in server."

whereas later in the file there's a bunch of support for it, and it appears to work fine.

Attachments:

* [http-docstring.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/047/047c4b5c885adb7132602f7a3c8c4eeb61d5e502/b794b71d49dac31478ec22b581e6eb82f18f381a.patch) (781 bytes) - added by exarkun on 2011-02-27 14:17:15Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__4911 4911
type__defect defect
reporter__andy andy
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web web
keywords__ 
time__1298587764000000 1298587764000000
changetime__1299679843000000 1299679843000000
version__None None
owner__exarkun exarkun
cc__jknight
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_3"></a>|@itamarst set owner to @exarkun|
|-|-|

1. Needs .misc news file.

2. The docstring isn't quite a accurate - some of the code is used by _newclient, e.g. the chunked encoding parser. So maybe mention "Plus some utility classes used by the HTTP client" (PotentialDataLoss is the only public thing a client *user* would end up knowing about, I think.)

3. It'd be nice if my name was removed as maintainer. Not essential though.

Other than that, looks good. Feel free to commit with [#11338](https://github.com/twisted/twisted/issues/11338), and #2/#3 if you feel like it.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set status to `closed`|
|-|-|

(In [30897]) Update the twisted.web.http module docstring to more accurately reflect that module's current state

Author: exarkun
Reviewer: itamar
Fixes: [#4911](https://github.com/twisted/twisted/issues/4911)
